INFO     root:test_03_syncManage.py:28 ==========开始执行用例：usersyncManage_list_01==========
INFO     root:runcase.py:29 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:45 重组后的用例信息：{'name': '同步用户列表状态查询', 'description': '同步用户列表状态查询', 'request': {'data': {'page': 1, 'rows': 20}, 'method': 'GET', 'address': '/api/user/v1/syncManage/list', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjQ1MDY4ZDc5LWMyYjgtNGUzMy1iMGJlLWRkMTBjZWU5NDA5NCIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI0NTA2OGQ3OS1jMmI4LTRlMzMtYjBiZS1kZDEwY2VlOTQwOTQiLCJleHAiOjE2ODcwODY4MTksImlhdCI6MTY4NjkwNjgxOX0.WweSsD34ofTCj16zOOE9g1_jAG7EDCQReO6jPG4pt4XnGdeXUHNfyGDkwq3Ln1lZNSG91uKBGAJ9LvxLzpHTLA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userPush'}}, 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '用户管理', 'feature': '用户同步管理-用户推送', 'story': '同步用户列表状态查询接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'usersyncManage_list_01'}
INFO     root:runcase.py:81 请求地址：https://192.168.148.174:31000/api/user/v1/syncManage/list
INFO     root:runcase.py:82 请求方法：GET
INFO     root:runcase.py:83 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjQ1MDY4ZDc5LWMyYjgtNGUzMy1iMGJlLWRkMTBjZWU5NDA5NCIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI0NTA2OGQ3OS1jMmI4LTRlMzMtYjBiZS1kZDEwY2VlOTQwOTQiLCJleHAiOjE2ODcwODY4MTksImlhdCI6MTY4NjkwNjgxOX0.WweSsD34ofTCj16zOOE9g1_jAG7EDCQReO6jPG4pt4XnGdeXUHNfyGDkwq3Ln1lZNSG91uKBGAJ9LvxLzpHTLA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userPush'}
INFO     root:runcase.py:93 请求参数：{'page': 1, 'rows': 20}
DEBUG    urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
DEBUG    urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "GET /api/user/v1/syncManage/list?page=1&rows=20 HTTP/1.1" 200 688
INFO     root:runcase.py:158 请求接口结果： {'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 3, 'list': [{'privateTopic': 'sync-private-1', 'systemName': '接入系统-1', 'syncGroupStatus': '72/72', 'syncUserStatus': '2356/2357', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-15 19:19:15', 'syncStatus': 1}, {'privateTopic': 'sync-private-approve002', 'systemName': '自定义审批系统', 'syncGroupStatus': '3/3', 'syncUserStatus': '2/2', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-08 18:38:03', 'syncStatus': 1}, {'privateTopic': 'sync-private-business002', 'systemName': '集中监控与审计系统', 'syncGroupStatus': '3/3', 'syncUserStatus': '2/2', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-08 18:37:56', 'syncStatus': 1}]}}
INFO     root:checkresult.py:111 code断言通过, 期望状态码:200, 实际状态码:200
INFO     root:checkresult.py:46 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 3, 'list': [{'privateTopic': 'sync-private-1', 'systemName': '接入系统-1', 'syncGroupStatus': '72/72', 'syncUserStatus': '2356/2357', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-15 19:19:15', 'syncStatus': 1}, {'privateTopic': 'sync-private-approve002', 'systemName': '自定义审批系统', 'syncGroupStatus': '3/3', 'syncUserStatus': '2/2', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-08 18:38:03', 'syncStatus': 1}, {'privateTopic': 'sync-private-business002', 'systemName': '集中监控与审计系统', 'syncGroupStatus': '3/3', 'syncUserStatus': '2/2', 'syncGroupCustomStatus': None, 'lastPushTime': '2023-06-08 18:37:56', 'syncStatus': 1}]}}
INFO     root:runcase.py:61 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-